### PR TITLE
Pass real type information to serialbox at init

### PR DESCRIPTION
**Pull request by clementval**
Get back the modification from Xavier made to support single/double precision. Taken from this branch https://github.com/lxavier/cosmo-pompa/tree/conv_tiedke_with_ser
Fix issue #61 
- add message if part of perturbation initialization is missing

@lxavier I took your changes from your branch. Can you review and merge the PR if it's fine for you ?

### CODE REVIEW

**Comment by clementval** [comment]
launch jenkins

**Comment by clementval** [comment]
launch jenkins

**Comment by clementval** [comment]
Just greina fails ... But as I have no access to it and it seems to be a problem of loading module, I think this is not problematic.

File: fortran/utils_ppser.f90 [hunk 1]
```diff
@@ -84,9 +87,9 @@ SUBROUTINE ppser_initialize(directory, prefix, mode, prefix_ref, mpi_rank, rprec
 
   ! Get data size
   intvalue = 0
-  realvalue = 0.
+  realvalue = 4 ! default real type value 
   ppser_intlength = INT(SIZE(TRANSFER(intvalue, buffer)))
-  ppser_reallength = INT(SIZE(TRANSFER(realvalue, buffer)))
+  ppser_reallength = realtype
```
Comment (lxavier) [inline fortran/utils_ppser.f90]: I think there was a bug in my branch, what happen if real type is not present (it is an optional argument)

Comment (clementval) [inline fortran/utils_ppser.f90]: Right, I'll fix that

**Comment by clementval** [comment]
launch jenkins

**Comment by clementval** [comment]
@lxavier So I removed the realvalue and set a default real type size when the optional value is not present

**Pull request merged**